### ISSUE TITLE
`gtk.EditableTextWidget` unavailable

### ISSUE THREAD

**Issue by sewnie** [diamondburned/gotk4-adwaita#10]
```
$ go build -v
...
github.com/diamondburned/gotk4-adwaita/pkg/adw
# github.com/diamondburned/gotk4-adwaita/pkg/adw
../../.cache/go/mod/github.com/diamondburned/gotk4-adwaita/pkg@v0.0.0-20231227234216-6f95643be092/adw/adw-entry-row_1_2.go:76:6: undefined: gtk.EditableTextWidget
../../.cache/go/mod/github.com/diamondburned/gotk4-adwaita/pkg@v0.0.0-20231227234216-6f95643be092/adw/adw-entry-row_1_2.go:140:27: undefined: gtk.EditableTextWidget
../../.cache/go/mod/github.com/diamondburned/gotk4-adwaita/pkg@v0.0.0-20231227234216-6f95643be092/adw/adw-spin-row_1_4.go:71:6: undefined: gtk.EditableTextWidget
../../.cache/go/mod/github.com/diamondburned/gotk4-adwaita/pkg@v0.0.0-20231227234216-6f95643be092/adw/adw-spin-row_1_4.go:137:27: undefined: gtk.EditableTextWidget
../../.cache/go/mod/github.com/diamondburned/gotk4-adwaita/pkg@v0.0.0-20231227234216-6f95643be092/adw/adw-password-entry-row_1_2.go:122:28: undefined: gtk.EditableTextWidget
```

**Comment by diamondburned** [comment]
This should be fixed with 96b1704606a8e9e66d11d9e3e43d5d1f7c08850f. Feel free to reopen it if you can still replicate it.

**Comment by sewnie** [comment]
thanks
